<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1095</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HashApp!</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../MonoalphabeticCipher/images.qrc">
    <normaloff>:/img/img/cryptography.svg</normaloff>:/img/img/cryptography.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">#frame{
	background:rgb(243, 246, 253);
	border-radius: 20px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLineEdit" name="MD5Value">
      <property name="geometry">
       <rect>
        <x>112</x>
        <y>340</y>
        <width>911</width>
        <height>31</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="SHA1Value">
      <property name="geometry">
       <rect>
        <x>112</x>
        <y>381</y>
        <width>911</width>
        <height>31</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="SHA256Value">
      <property name="geometry">
       <rect>
        <x>112</x>
        <y>421</y>
        <width>911</width>
        <height>31</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="TextString">
      <property name="geometry">
       <rect>
        <x>378</x>
        <y>129</y>
        <width>341</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="HexString">
      <property name="geometry">
       <rect>
        <x>378</x>
        <y>169</y>
        <width>341</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>377</x>
        <y>210</y>
        <width>343</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#pushButton {
background-color: rgb(204, 180, 254);
border-radius: 0px;
color: rgb(255, 255, 255);
}
QPushButton#pushButton:hover {
background-color: rgb(79, 84, 188);
}</string>
      </property>
      <property name="text">
       <string>Choose File</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>129</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Text String</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>283</x>
        <y>174</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Hex String</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>220</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>File</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>32</x>
        <y>344</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>MD5</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>388</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>SHA-1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>426</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>SHA-256</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="file">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>260</y>
        <width>641</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="SHA384Value">
      <property name="geometry">
       <rect>
        <x>112</x>
        <y>465</y>
        <width>911</width>
        <height>31</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>470</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>SHA-384</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="SHA512Value">
      <property name="geometry">
       <rect>
        <x>112</x>
        <y>507</y>
        <width>911</width>
        <height>31</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>512</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>SHA-512</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="SHA3256Value">
      <property name="geometry">
       <rect>
        <x>112</x>
        <y>551</y>
        <width>911</width>
        <height>31</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>556</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>SHA3-256</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="SHA3512Value">
      <property name="geometry">
       <rect>
        <x>112</x>
        <y>595</y>
        <width>911</width>
        <height>31</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>600</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>SHA3-512</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btnClose">
      <property name="geometry">
       <rect>
        <x>1050</x>
        <y>10</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 8px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../MonoalphabeticCipher/images.qrc">
        <normaloff>:/img/img/close.svg</normaloff>:/img/img/close.svg</iconset>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>20</y>
        <width>431</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins SemiBold</family>
        <pointsize>28</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(74, 74, 74);</string>
      </property>
      <property name="text">
       <string>HashApp!</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../MonoalphabeticCipher/images.qrc"/>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
